### DISCUSSION TITLE
Weird highlights

### DISCUSSION THREAD

**Discussion by lepinvlad** [LazyVim/LazyVim#4636] [Q&A] [answered]
Lazyvim is perfect but I don't know what it is and how to disable it

Check the photo I uploaded

Does someone know how to disable them?
<img width="495" alt="Снимок экрана 2024-10-23 в 19 35 55" src="https://github.com/user-attachments/assets/456a23d3-94c8-4c24-9f32-53e324e94661">

**Comment by dpetka2001** [comment, marked as answer]
You probably used a snippet to insert this text. If that's the case, then press `<Tab>` until you get out of snippet range. Unfortunately, this is a current side effect of Neovim's builtin `vim.snippet` engine if I'm not mistaken (that highlights the whole snippet region instead of just the placeholders).

**Reply by lepinvlad** [reply]
Thanks for the answer!

**Comment by dotfrag** [comment]
Duplicates https://github.com/LazyVim/LazyVim/discussions/4632.
